modules/router/src/test/collateral.spec.ts

```diff
@@ -425,6 +504,32 @@ describe(testName, () => {
         expect(node.reconcileDeposit.callCount).to.be.eq(1);
       });
 
+      it("if the profile.target is 0 and a requestedAmount is supplied", async () => {
```

Comment:
Would like to see this test name being more descriptive, to say `should do something`.

Reply:
its nested in the `should work` describe